### PR TITLE
Adding OSP13 compatiable nic-configs

### PR DESCRIPTION
**Pull request by akrzos**

### CODE REVIEW

File: roles/overcloud-prepare-templates/files/nic-configs/13/private/1029u-cephstorage.yaml [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+    description: The subnet CIDR of the control plane network.
+    type: string
+  ControlPlaneDefaultRoute: # Override this via parameter_defaults
+    description: The default route of the control plane network.
+    type: string
+  ExternalInterfaceDefaultRoute: # Not used by default in this template
+    default: 10.0.0.1
+    description: The default route of the external network.
+    type: string
+  ManagementInterfaceDefaultRoute: # Commented out by default in this template
```
Comment (mbruzek) [inline roles/overcloud-prepare-templates/files/nic-configs/13/private/1029u-cephstorage.yaml]: When comparing the 11/1029p-controller.yml and 13/1029p-controller.yml this is a new route. Is this by design?

Comment (mbruzek) [reply roles/overcloud-prepare-templates/files/nic-configs/13/private/1029u-cephstorage.yaml]: The "BondInterfaceOvsOptions" section was only in the 11/1029p-controller.yml file and not present in the 13/1029p-controller.yml file.

Comment (akrzos) [reply roles/overcloud-prepare-templates/files/nic-configs/13/private/1029u-cephstorage.yaml]: AIUI these are just default settings.  I think the differences rest from the origin of the nic-configs (The OSP11 ones pre-date OSP11 by far considering the date is April 2015 (Ocata was released in 2017)).  Now that we maintain 4 sets of nic-configs we would definitely see any differences that have surfaces since the others have been generated.  The OSP13 nic-configs have their defaults "header" copied directly out of example configs off the last OSP11 build.  

In short this doesn't affect our configuration but rather just "better" matches what our install places on the Undercloud.

**Comment by akrzos** [comment]
We now have a successful deploy with OSP13 nic-configs.

**Comment by mbruzek** [comment]
I see a successful deploy of OpenShift 3.10 on the OpenStack, +1 to merge this change.
